assets/javascripts/discourse/initializers/global-filter-preference.js

```diff
@@ -223,4 +223,8 @@ function setFilteredCategoriesForGlobalFilter(api) {
   api.modifySelectKit("category-drop").appendContent(() => {
```

Comment:
Do we still need the `appendContent` call?

Reply:
We do need to update the `category-drop` components content. In it's case we can use `appendContent` or `replaceContent` with the same result.